packages/models-library/src/models_library/api_schemas_webserver/projects.py

```diff
@@ -83,11 +83,11 @@ class ProjectGet(OutputSchema):
     quality: dict[str, Any] = Field(
         default_factory=dict, json_schema_extra={"default": {}}
     )
-    dev: dict | None
+    dev: dict | None = None
     permalink: ProjectPermalink | None = None
-    workspace_id: WorkspaceID | None
-    folder_id: FolderID | None
-    trashed_at: datetime | None
+    workspace_id: WorkspaceID | None = None
+    folder_id: FolderID | None = None
+    trashed_at: datetime | None = None
 
```

Comment:
thes changes look suspicious. this is an output and you made them now optional which they were not. can you double check please?